plugins/uefi-capsule/fu-plugin-uefi-capsule.c

```diff
@@ -41,6 +41,7 @@ fu_plugin_init (FuPlugin *plugin)
 	fu_plugin_add_rule (plugin, FU_PLUGIN_RULE_METADATA_SOURCE, "tpm");
 	fu_plugin_add_rule (plugin, FU_PLUGIN_RULE_METADATA_SOURCE, "tpm_eventlog");
 	fu_plugin_add_rule (plugin, FU_PLUGIN_RULE_METADATA_SOURCE, "dell");
+	fu_plugin_add_rule (plugin, FU_PLUGIN_RULE_METADATA_SOURCE, "linux_lockdown");
```

Comment:
I can see how this matters for flashrom and superio, but not for uefi capsule.  Care to elaborate?

Reply:
Selection bias; if lockdown is enabled the superio update won't be offered and we'll only get reports where the machine is not locked down.

Reply:
So it's trying to get reports of machines that use uefi capsule + superio, but have lockdown enabled?  If that's going to be a big combination for the future the superio plugin should probably be ported to run in UEFI instead.

Reply:
> So it's trying to get reports of machines that use uefi capsule + superio, but have lockdown enabled

Exactly that.

> the superio plugin should probably be ported to run in UEFI instead

That's my plan. I might have some more headcount to work on fwupd in the future, and I thought this would be a good self-contained task. But before that, I need justification :)

Reply:
OK..